master/reference/calicoctl/resources/globalnetworkpolicy.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Global Network Policy Resource (GlobalNetworkPolicy)
+---
+
+A Global Network Policy resource (GlobalNetworkPolicy) represents an ordered set of rules which are applied
+to a collection of endpoints which match a [label selector](#selector).
+
+GlobalNetworkPolicy is not a namespaced resource, which means, GlobalNetworkPolicy applies to [WorkloadEndpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint) in all namespaces. 
+See [NetworkPolicy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy) for namespaced Network Policy.
+
+GlobalNetworkPolicy resources can be used to define network connectivity rules between groups of Calico endpoints and host endpoints, and
+take precedence over [Profile resources]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/profile) if any are defined.
+
+For `calicoctl` commands that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `globalnetworkpolicy`, `globalnetworkpolicies`, `gnp`, `gnps`.
```

Comment:
A sample command would be nice to show what you mean

Reply:
the scope of these docs is only to be the single source of truth for all our resources, calicoctl command examples for these are in the `commands` section (I'll update them in a separate PR)